package.json

```diff
@@ -365,7 +365,8 @@
     "web3-core-method": "patch:web3-core-method@npm:1.6.0#.yarn/patches/web3-core-method-npm-1.6.0-39b7ee99f4.patch",
     "bignumber.js": "9.1.1",
     "lottie-react-native": "patch:lottie-react-native@npm:5.1.5#.yarn/patches/lottie-react-native-npm-5.1.5-384eb12f33.patch",
-    "@usedapp/core": "patch:@usedapp/core@npm:1.2.6#.yarn/patches/@usedapp-core-npm-1.2.6-8dbe982e03.patch"
+    "@usedapp/core": "patch:@usedapp/core@npm:1.2.6#.yarn/patches/@usedapp-core-npm-1.2.6-8dbe982e03.patch",
+    "@gooddollar/goodprotocol@2.0.5-beta.7": "patch:@gooddollar/goodprotocol@npm%3A2.0.5-beta.7#./.yarn/patches/@gooddollar-goodprotocol-npm-2.0.5-beta.7-5be413b3e1.patch"
```

Comment:
remove this and the patch

Reply:
this patch is needed because  "development-goerli":  section at '@gooddollar/goodprotocol/releases/deployment.json' have no networkId value been set

once you'll fix this and release new beta package - i will remove patch